trackma/lib/libkitsu.py

```diff
@@ -549,29 +565,36 @@ def _parse_info(self, media):
             'image_thumb': attr['posterImage'] and attr['posterImage']['tiny'],
             'start_date':  self._str2date(attr['startDate']),
             'end_date':    self._str2date(attr['endDate']),
-            'type':        self.type_translate.get(attr['subtype'],utils.TYPE_UNKNOWN),
-            'status':      self.status_translate.get(attr['status'],utils.STATUS_UNKNOWN),
+            'type':        self.type_translate.get(attr['subtype'], utils.TYPE_UNKNOWN),
+            'status':      self.status_translate.get(attr['status'], utils.STATUS_UNKNOWN),
             'url': "https://kitsu.io/{}/{}".format(self.mediatype, attr['slug']),
             'aliases':     list(filter(None, attr['titles'].values())),
             'extra': [
-                ('Synopsis', attr['description']),
-                ('Type',     attr['subtype']),
-                ('Titles',  list(filter(None, attr['titles'].values())) ),
-                ('Average Rating', attr['averageRating'] or '?'),
-                ('Popularity Rank', attr['popularityRank']),
-                ('Rating Rank', attr['ratingRank']),
-                ('Age Rating', attr['ageRating']),
+                ('Synopsis',            attr['description']),
+                ('Type',                attr['subtype']),
+                ('Titles',              list(
+                    filter(None, attr['titles'].values()))),
+                ('Synonyms',            attr.get('abbreviatedTitles', '?')),
+                ('Average Rating',      attr.get('averageRating', '?')),
+                ('Rank',                "Popularity #{}\nRating #{}".format(
+                    attr.get('popularityRank', '?'), attr.get('ratingRank', '?'))),
+                ('Age Rating',          "{} ({})".format(
+                    attr.get('ageRating', 'Unknown'), attr.get('ageRatingGuide', 'Unknown'))),
             ]
         })
 
+        if attr.get('nsfw', False):
+            info['extra'].insert(
+                2, ('NSFW', 'Yes'))
```

Comment:
Why do you wrap lines so aggressively?

Reply:
[autopep8](https://github.com/ahmubashshir/dotfiles/blob/master/dot-config:git/hooks/hooks.d/pre-commit/format)'s fault :3

Reply:
I suggest to disable such a tool for pull requests to other projects, as this introduces uncecessary and often undesired changes, not unlike the one I highlighted above. It has already caused me to manually resolve conflicts in an open PR of mine in multiple files and was quite a bother.

This isn't my project, so I'm not going to campaign for or against certain code style choices, but I consider a maximum line length below 100 to be counterproductive, and doubly so if the line breaks are done arbitrarily and without a cohesive pattern (and on a line with a width of 52). If you want automatic formatting, `black` is most likely the better choice.

Reply:
ok, I'll set it to run if it's explicitly enabled.